src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -1810,6 +1810,65 @@
                         </Label.ToolTip>
                     </Label>
                 </Grid>
+                <Popup Name="popupMessage" Height="140" Width="472" StaysOpen="True" Placement="RelativePoint" VerticalOffset="-70" HorizontalOffset="180" AllowsTransparency="True">
```

Comment:
Sorry, do we still need these code?

Reply:
@Jingyi-Wen Do you think there will be some designs around toast notification inside preferences panel soon? If not, I would like to not have the popup for now

Reply:
> @Jingyi-Wen Do you think there will be some designs around toast notification inside preferences panel soon? If not, I would like to not have the popup for now

I agree we can have the popup for now.

Reply:
@QilongTang  Done